File: java64/nivel1/exer2.java
import java.util.Scanner;

public class exer2 {
    public static void main(String[] args) {
        double cotacaoDolar;
        double valorDolar;
        double conversaoReal;

        Scanner input = new Scanner(System.in);
        System.out.println("Cotação do dolar: ");
        cotacaoDolar = input.nextDouble();
        System.out.println("Valor em dolar: ");
        valorDolar = input.nextDouble();

        conversaoReal = valorDolar * cotacaoDolar;

        System.out.println("Valor em real: " + conversaoReal);

        input.close();
    }
}

File: pyhton64/nivel1/exer2.py
cotacao_dolar = input('Cotação do dolar: ')
cotacao_dolar = float(cotacao_dolar)

valor_dolar = input('Valor em dolares: ')
valor_dolar = float(valor_dolar)

conversao_real = float(valor_dolar * cotacao_dolar)
print(f'Conversão: {conversao_real}')